docs/howto/openstack/cinder/retype-volumes.md

```diff
@@ -0,0 +1,134 @@
+---
+description: Changing the type of a volume ("retyping") is an offline operation that requires detaching the volume from its server, and setting its new type.
+---
+# Changing a volume's type
+
+You may occasionally need to change the type of a volume.
+This may be due to a volume type being phased out on {{brand}}'s part, necessitating data migration.
+Or you might want to enable or disable [volume-level encryption](encrypted-volumes.md).
+
+{{page.meta.description}}
+The resulting downtime may be quite substantial, particularly for large volumes.
+As such, when you need to retype volumes, you should plan ahead well in advance.
+
+## Prerequisites
+
+In order to retype volumes, you must use the OpenStack CLI, so make sure you [have it enabled](../../getting-started/enable-openstack-cli.md).
+
+If you are about to retype a large volume, or one that holds data associated with a critical service, you may be interested in an estimate of how long the retype operation will take.
+In that case, please file a support request with our [{{support}}](https://{{support_domain}}).
+
+## Checking the volume's state
+
+Assume you have a volume named `testvol` that is currently attached to a server named `testsrv`:
+
+```console
+$ openstack volume list --long
++----------------+---------+--------+------+---------+----------+----------------+------------+
+| ID             | Name    | Status | Size | Type    | Bootable | Attached to    | Properties |
++----------------+---------+--------+------+---------+----------+----------------+------------+
+| e233e7f3-f33b- | testvol | in-use |   50 | default | false    | Attached to    |            |
+| 4d7a-8f5b-785b |         |        |      |         |          | testsrv on     |            |
+| 34f670bf       |         |        |      |         |          | /dev/vdb       |            |
++----------------+---------+--------+------+---------+----------+----------------+------------+
+```
+
+In this example, this volume status is `in-use`, meaning it is currently attached to a server, and the volume type is `default`.
+
+## Detaching the volume
+
+You cannot retype a volume while it is attached to a server.
+You must thus detach it first.
```

Comment:
Should we say that root volumes can't be detached?

Reply:
Please feel free to send a longer patch, _after_ this one lands, explaining how to deal with BFV and retyping. That probably belongs in the Background section too, not in a how-to guide.